WordPress/jetpack_metadata/release_notes.txt

```diff
@@ -1,10 +1,7 @@
-* [*] Fixed email app detection in signup for Android 11, and removed the useless button when none are available. [https://github.com/wordpress-mobile/WordPress-Android/pull/15911]
```

Comment:
@ashiagr does that mean that I was wrong removing that entry then?

Reply:
I think we can remove this entry from release notes for JP as signup was never enabled until now and JP users need not know if anything was fixed related to signup.